ansible/ANSIBLE.md

```diff
@@ -0,0 +1,358 @@
+# Inventory
+I use a dynamic AWS inventory (from Bonus task). To be able to use it you need to install `aws_ec2` plugin, `boto3` library and 
+`geerlingguy.docker` collections.
+```shell
+pip install boto3
+ansible-galaxy install geerlingguy.docker
+ansible-galaxy collection install amazon.aws
+```
+
+In the yaml file `inventory/aws_ec2.yml` you can add filters or grouping. The `hostname` section is added to avoid
+hosts duplicating (my ec2 has public and private IPs, so it's kinda two hosts).
+```yaml
+plugin: aws_ec2
+
+regions:
+  - eu-north-1
+
+hostnames:
+  - name: dns-name
+    prefix: ''
+```
+
+Also, you can set inventory as default in `ansible.cfg`
+```
+[defaults]
+host_key_checking = False
```

Comment:
This one is cool btw

Anyway, fuck `ansible.cfg`, all my homies use group and host vars